src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IDynamicInvocationExpression.vb

```diff
@@ -0,0 +1,459 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+Imports Microsoft.CodeAnalysis.Test.Utilities
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.UnitTests.Semantics
+
+    Partial Public Class IOperationTests
+        Inherits SemanticModelTestBase
+        <CompilerTrait(CompilerFeature.IOperation)>
+        <Fact()>
+        Public Sub DynamicInvocationExpression_Basic()
+            Dim source = <![CDATA[
+Option Strict Off
+
+Class C
+    Private Sub M(c As Object, d As Object)
+        c.M2(d)'BIND:"c.M2(d)"
+    End Sub
+End Class]]>.Value
+
+            Dim expectedOperationTree = <![CDATA[
+IDynamicInvocationExpression (OperationKind.DynamicInvocationExpression, Type: System.Object) (Syntax: 'c.M2(d)')
+  Expression: IDynamicMemberReferenceExpression (Member Name: "M2", Containing Type: null) (OperationKind.DynamicMemberReferenceExpression, Type: System.Object) (Syntax: 'c.M2')
+      Type Arguments(0)
+      Instance Receiver: IParameterReferenceExpression: c (OperationKind.ParameterReferenceExpression, Type: System.Object) (Syntax: 'c')
+  ApplicableSymbols(0)
+  Arguments(1):
+      IParameterReferenceExpression: d (OperationKind.ParameterReferenceExpression, Type: System.Object) (Syntax: 'd')
+  ArgumentNames(0)
+  ArgumentRefKinds(0)
```

Comment:
>ArgumentRefKinds(0) [](start = 2, length = 19)

I would expect None vs. an empty set. There should be clear difference between VB output and C# output